### PR TITLE
New Button Sample port

### PR DESCRIPTION
**Pull request by officialgio**
This is the new button sample that's being ported to .NET 6. The project just shows the functionality of a button so it doesn't have much code.

### CODE REVIEW

**Comment by alirong** [comment]
这是来自QQ邮箱的自动回复邮件。
 
您好，我已经收到您的邮件。谢谢

File: Button/Button/MainActivity.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+        // Create your application here
+        Android.Widget.Button button = new (this);
+
+        button.Text = string.Format ("{0} clicks!!", count);
+        button.Click += delegate { button.Text = string.Format ("{0} clicks!!", ++count); };
+
+        SetContentView (button);
+    }
+}
```
Comment (tj-devel709) [inline Button/Button/MainActivity.cs:20]: Don't forget a new line for your cs files

**Review by tj-devel709 — CHANGES_REQUESTED** [review]
Only one comment here for me :)

**Review by rachelkang — APPROVED** [review]
LGTM! Builds and deploys as expected. Great work :)

File: Button/Button/MainActivity.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    int count = 0;
+
+    protected override void OnCreate (Bundle? bundle)
+    {
+        base.OnCreate (bundle);
+
+        // Create your application here
+        Android.Widget.Button button = new (this);
+
+        button.Text = string.Format ("{0} clicks!!", count);
```
Comment (jonathanpeppers) [inline Button/Button/MainActivity.cs:15]: This sample was probably written before we had the string interpolation C# feature, can you update the above to:
```suggestion
        button.Text = $"{count} clicks!!";
```
Then one other example below. Thanks!

**Comment by rachelkang** [comment]
Could you actually update the screenshot when you get a chance? Thanks :)

File: Button/Button/MainActivity.cs (hunk 2 above)
Comment (officialgio) [reply Button/Button/MainActivity.cs]: Hi Jonathan, I've just updated the code!

File: Button/Button/MainActivity.cs [hunk 3]
```diff
@@ -12,7 +12,7 @@ protected override void OnCreate (Bundle? bundle)
         // Create your application here
         Android.Widget.Button button = new (this);
 
-        button.Text = string.Format ("{0} clicks!!", count);
+        button.Text = $"{count} clicks!!";
         button.Click += delegate { 
             button.Text = string.Format ("{0} clicks!!", ++count);
```
[the remark this answers is not in the document]
Comment (officialgio) [reply Button/Button/MainActivity.cs]: Oops, I fixed it!

**Review by jonathanpeppers — APPROVED** [review]

**Pull request merged**